docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "2"
```

Comment:
Is there a specific reason you chose version 2 instead of 3?

Reply:
Switch to 3

Reply:
Actually, switch back to 2.3 to get the runtime option.